packages/roosterjs-editor-api/lib/format/setListNumberPosition.ts

```diff
@@ -0,0 +1,22 @@
+import { ChangeSource, IEditor } from 'roosterjs-editor-types';
+
+/**
+ * Set the List Number Position (Margin) of a List
+ * @param editor The editor instance
+ * @param list The HTML element
+ * @param numberPosition The number position value to be set to the list
+ * @param unit The Unit to be used.
+ */
+export default function setListNumberPosition(
```

Comment:
Need a better name, this does not only work for numbering list, but it actually works for all block elements.